Makefile

```diff
@@ -0,0 +1,35 @@
+DOCKER       = docker
+HUGO_VERSION = 0.40.3
+DOCKER_IMAGE = client-go-docs
+DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(PWD):/src
+
+.PHONY: all build sass build-preview help serve
+
+help: ## Show this help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
```

Comment:
This is some serious make-magic... I am not going to pretend I know what this does, but I guess it extracts out the comments after each Make target to print it out nicely? Clever 😄